### PR TITLE
Login Functionalty Test Automation 

### PR DESCRIPTION
**Pull request by Ozleblebici**
Local test are done, ready for review 
image

### CODE REVIEW

**Comment by Ozleblebici** [comment]
Test comment

**Comment by Ozleblebici** [comment]
@Tirkish89 could you please review my PR ?

File: src/test/java/LoginFuncTest.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+public class LoginFuncTest {
+
+    public static void main(String[] args) {
+        System.out.println("navigate to login page");
+        System.out.println("enter username");
+        System.out.println("enter password");
+        System.out.println("click login");
+        System.out.println("verify login page!!");
```
Comment (ATABEG00) [inline src/test/java/LoginFuncTest.java:8]: add more lines

**Review by ATABEG00 — COMMENTED** [review]
nice project

**Review by ATABEG00 — APPROVED** [review]
.

**Review by Tirkish89 — COMMENTED** [review]
good job.

**Review by 1erkan1 — APPROVED** [review]
good job

**Review by Tirkish89 — APPROVED** [review]
good job.

**Review by Gylych1110 — APPROVED** [review]
Good job

File: src/test/java/Test1.java
Comment (Tirkish89) [inline src/test/java/Test1.java:1]: reviewed.

**Review by Tirkish89 — APPROVED** [review]

**Comment by Gujalwar** [comment]
looks good to me.

**Pull request merged**